apps/doop/src/App.tsx

```diff
@@ -7,6 +7,7 @@ import React, { useLayoutEffect } from "react"
 
 import { AppShellProvider, ContentHeading } from "@cloudoperators/juno-ui-components"
 import AppContent from "./components/AppContent"
+// @ts-expect-error TS(2792) FIXME: Cannot find module './styles.scss?inline'. Did you... Remove this comment to see the full error message
```

Comment:
Please [reference](https://github.com/cloudoperators/juno/blob/326b01e63be27508844fa9bc187f474bead43206/apps/supernova/src/types/index.d.ts#L6) `vite/client` types like we did in `supernova`.